src/libs/sqliteHandler.go

```diff
@@ -0,0 +1,748 @@
+package libs
+
+import (
+	"database/sql"
+	"encoding/json"
+	"strconv"
+	"strings"
+	"time"
+	"fmt"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/accuknox/auto-policy-discovery/src/types"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+const WorkloadProcessFileSetSQLite_TableName = "workload_process_fileset"
+const TableNetworkPolicySQLite_TableName = "network_policy"
+const TableSystemPolicySQLite_TableName = "system_policy"
+
+// ================ //
+// == Connection == //
+// ================ //
+
+var MockSqlite sqlmock.Sqlmock = nil
+var MockDBSQLite *sql.DB = nil
+
+func NewMockSQLite() (*sql.DB, sqlmock.Sqlmock) {
+	db, mock, err := sqlmock.New()
+	if err != nil {
+		log.Error().Msgf("an error '%s' was not expected when opening a stub database connection", err)
+	}
+
+	MockSqlite = mock
+	MockDBSQLite = db
+
+	return db, mock
+}
+
+func waitForDBSQLite(db *sql.DB) {
+	for {
+		err := db.Ping()
+		if err != nil {
+			time.Sleep(time.Second * 1)
+			log.Error().Msgf("db.Ping() failed. Will retry. err=%s", err.Error())
+		} else {
+			break
+		}
+	}
+}
+
+func connectSQLite(cfg types.ConfigDB) (db *sql.DB) {
+	if MockDB != nil {
+		return MockDB
+	}
+
+	dbconn := cfg.DBUser + ":" + cfg.DBPass + "@tcp(" + cfg.DBHost + ":" + cfg.DBPort + ")/" + cfg.DBName
+	fmt.Println(dbconn)
+	//db, err := sql.Open(cfg.DBDriver, dbconn)
+	db, err := sql.Open(cfg.DBDriver, "/home/vagrant/accuknox.db")
```

Comment:
@ErenYeager713 Please do not hard-code any paths. Get this path from local-file config and use the same across the code.

Reply:
Okay